pkg/app/piped/notifier/slack.go

```diff
@@ -283,3 +298,7 @@ func getAccountsAsString(accounts []string) string {
 	}
 	return strings.Join(formattedAccounts, " ")
 }
+
+func makeCommitLink(gp *model.ApplicationGitPath, hash string) string {
+	return strings.Replace(gp.Url, fmt.Sprintf("/tree/master/%s", gp.Path), "", 1) + fmt.Sprintf("/commit/%s", hash)
```

Comment:
Ref isn't always "master"

Reply:
If you want to build the commit link, `git.MakeCommitURL` is likely to be for you.

https://github.com/pipe-cd/pipe/blob/40ee21ed2530ce5c709a8b6a63326a7f6dfc0e06/pkg/git/url.go#L25

Reply:
Thanks, I'll update it, too.

Reply:
ff81c6f